### PR TITLE
Enhanced visit web

### LINKED ISSUE #79: Standardise CommandResult displayed
- `addReview` shows 'Review added to: [Restaurant Name]'
- `goTo` shows 'Visited Website of Restaurant: [Index]'

### LINKED ISSUE #93: As a user, I want to visit website of restaurants that are not in my restaurant list
so that I can get the information of the restaurant to be added into my visit restaurant list

### PR DESCRIPTION
**Pull request by Wklee96**
fixes #79, #92, #93, #99 

- [X] Prevent user from entering any weblink that is invalid
- [X] Added new visitWeb functionality, to visit website of any weblink input by user
- [x] If weblink of Restaurant is closed, and visitWeb is unable to load, there should be an error message
- [ ] Remove review list panels when visitWeb is called

### CODE REVIEW

**Comment by Wklee96** [comment]
Will be pushing **Remove review list panels when visitWeb is called** to v1.4

**Comment by skpai27** [comment]
Some test cases should be added for automatic testing! This would detect any bugs that arise when future changes are made to the code and other functionalities that could indirectly or directly affect this part. Perhaps it is possible to include a test where an invalid URL is given, and also one where a valid URL is given.

**Review by skpai27 — APPROVED** [review]
I think these changes are valid, and that there is good handling of edge cases. However, as I have stated before, it would be prudent to include automated test cases to detect any errors that could arise after further modifications to the project.

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 1]
```diff
@@ -199,12 +200,27 @@ public static Weblink parseWeblink(String weblink) throws ParseException {
         if (!Weblink.isValidWeblinkString(trimmedWeblink)) {
             throw new ParseException(Weblink.MESSAGE_CONSTRAINTS);
         }
-        if (!Weblink.isValidWeblinkUrl(trimmedWeblink)) {
-            throw new ParseException(Weblink.INVALID_URL_MESSAGE);
+
+        try {
+            checkUrl(trimmedWeblink);
+        } catch (NoInternetException e) {
```
Comment (chanqingzhou) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java:206]: Why after checking url do you throw a noInternetException

**Review by chanqingzhou — APPROVED** [review]
No internetException should not be thrown if url is not valid. Otherwise, its fine
